### PR TITLE
Revert to encrypted buildconfig for docs

### PR DESCRIPTION
**Pull request by accorvin**
I was unable to get the buildconfig trigger working with the secret
reference, so this puts the secret values back into the buildconfig,
requiring that the file be sops encrypted.

### CODE REVIEW

File: datahub-help/base/stasis-reactor-build-config.enc.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+spec:
+    output:
+        to:
+            kind: ImageStreamTag
+            name: stasis-reactor:latest
+    source:
+        type: Git
+        git:
+            ref: 0.2.5
+            uri: https://gitlab.com/elmiko/stasis-reactor.git
```
Comment (maulikjs) [inline datahub-help/base/stasis-reactor-build-config.enc.yaml:16]: @accorvin should be clone this repo under the DH org?

Comment (accorvin) [reply datahub-help/base/stasis-reactor-build-config.enc.yaml:16]: I don't know that it's worth the time to be honest. We never change the code there, and I think we'll eventually move away from using the stasis-reactor tool to using something else (@anishasthana did a POC with some other tool but I forget what it's called)

Comment (anishasthana) [reply datahub-help/base/stasis-reactor-build-config.enc.yaml:16]: I agree re: forking. It doesn't make sense to have another clone of the repo.

**Review by anishasthana — COMMENTED** [review]
/lgtm

File: datahub-help/base/stasis-reactor-build-config.enc.yaml (hunk 1 above)
Comment (maulikjs) [reply datahub-help/base/stasis-reactor-build-config.enc.yaml:16]: Sgtm

**Pull request merged**